pom.xml

```diff
@@ -60,6 +60,18 @@ limitations under the License.
       <artifactId>google-cloud-datastore</artifactId>
       <version>1.52.0</version>
     </dependency>
+    
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-language</artifactId>
+      <version>1.55.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative</artifactId> <!-- Use 'netty-all' for 4.0 or above -->
+      <version>2.0.9.Final</version>
```

Comment:
There seem to be newer versions of this library: https://mvnrepository.com/artifact/io.netty/netty-tcnative

Or here? https://mvnrepository.com/artifact/io.netty/netty-all

Do we need a specific version, or can we go with the latest and greatest?

Reply:
We specifically need `netty-tcnative`, I tried `netty-all` before and it didn't work., at least not for me.

As for the version -- turns out I wasn't paying attention and just grabbed the bottom one from https://repo.maven.apache.org/maven2/io/netty/netty-tcnative/, but there's actually a more recent one higher up due to the sort. I'll switch it out and see if it works just as well.